Configuration/StandardSequences/python/AlCaRecoStreams_cff.py

```diff
@@ -175,6 +175,7 @@
 pathALCARECOSiStripCalMinBiasAAG = cms.Path(seqALCARECOSiStripCalMinBiasAAG*ALCARECOSiStripCalMinBiasAAGDQM)
 pathALCARECOSiStripCalSmallBiasScan = cms.Path(seqALCARECOSiStripCalSmallBiasScan)
 pathALCARECOSiStripCalZeroBias = cms.Path(seqALCARECOSiStripCalZeroBias*ALCARECOSiStripCalZeroBiasDQM)
+pathALCARECOSiStripCalZeroBias.replace(consecutiveHEs, unpackTcds*consecutiveHEs)
```

Comment:
Why not modifying the sequence containing `consecutiveHEs` direclty?

Reply:
same comment as https://github.com/cms-sw/cmssw/pull/29116#discussion_r388455489

Reply:
I thought it was better to replace that at runtime only for AlCaRecosStreams_cff.py.

Can you please confirm it is better to modify the original sequence directly?

Reply:
I have now seen comment https://github.com/cms-sw/cmssw/pull/29116#discussion_r388455489 . Fixed (will push in a while)